setup.py

```diff
@@ -16,7 +16,6 @@
     'numpy',
     'imageio',
     'pillow',
-    'appdirs',
```

Comment:
Is this removal related?

Reply:
This should have happened in https://github.com/pyvista/pyvista/pull/3218